From 26416e6c4ab0c9720157ac8885880a5d7980179e Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Fri, 20 Mar 2015 14:00:04 +0800
Subject: [PATCH 0538/1594] MLK-10455-2 ARM: dts: imx7d: add ddrc node support

commit b8850adba55d0c620e50567872412bfade5037a5 from
git://git.freescale.com/imx/linux-2.6-imx.git

Add DDRC dts support for DDRC driver.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 49e870d..63c85c3 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -755,6 +755,11 @@
 				reg = <0x30760000 0x10000>;
 				status = "disabled";
 			};
+
+			ddrc: ddrc@307a0000 {
+				compatible = "fsl,imx7-ddrc";
+				reg = <0x307a0000 0x10000>;
+			};
 		};
 
 		aips3: aips-bus@30800000 {
-- 
1.7.5.4

